modules/ROOT/pages/deployment/container/orchestration/tab-pages/breaking-changes.adoc

```diff
@@ -0,0 +1,62 @@
+= Helm Chart Breaking Changes
+:toc: right
+
+////
+Note that there is only this one master file for all breaking changes.
+For each breaking changes block you need:
+* a new section on top of the lastest one. (actual first, former second)
+* directly above the section an ID (see below how it is done) with the corresponding version equal to the version name so it can be accessed via an anchor. This can not be automated and must be maintained manually.
```

Comment:
```suggestion
* directly above the section an ID (see below how it is done) with the corresponding version equal to the version name so it can be accessed via an anchor. This cannot be automated and must be maintained manually.
```